### ISSUE TITLE
Learn hex codes differ each button press

### ISSUE THREAD

**Issue by wizardofozzie** [lprhodes/homebridge-broadlink-rm#752]
I have managed to enable the `learn` button on the Homebridge (raspberry pi 4). 

I then push the on/off button on the device (Bonaire split system heater/cooler) and grab the hex value from the log. I repeat for the same button and get a different hex value. I repeat that so that I have 8 values (a,b,c,d,e,f,g,h) of hex values, of which 4 (if off/on toggle) or 8 of those values should be identical. They are all the same.

TL;DR- the same button pushed 8 times under learn gives 8 different hex values
